src/content/concepts/index.md

```diff
@@ -151,7 +151,7 @@ Using plugins in your webpack config is straightforward - however, there are man
 
 ## Mode
 
-By setting the `mode` parameter to either `development` or `production`, you can enable webpack's built-in optimizations that correspond with the selected mode.
+By setting the `mode` parameter to either `development` or `production`, you can enable webpack's built-in optimizations that correspond with the selected mode. If you would like to opt out of these optimizations, you may pass `none` instead.
```

Comment:
@Tvrqvoise could you move this line beloe the *Mode: none* heading? I think it makes more sense there, if not, having just a title with a code snippet doens't add enough valuable content in my opinion.

Reply:
Sure man